src/app/ngrx/effects/routing.effects.ts

```diff
@@ -31,32 +24,21 @@ export class RoutingEffects {
   @Effect()
   customRouterNavigation$: Observable<Action> = this.actions$.pipe(
     ofType(ROUTER_NAVIGATION),
-    map((action: ACTIONS.CustomRouterNavigationAction) => action.payload),
-    switchMap((payload: ACTIONS.CustomRouterNavigationPayload) => {
-      const routerState: RouterModel = {...payload.routerState};
+    map((action: RouterNavigationAction) => action.payload),
+    switchMap((payload: RouterNavigationPayload<any>) => {
```

Comment:
This change is just me reading the docs/source more closely on what incoming types these are.
The ROUTER_NAVIGATION action comes from the ngrx/router-store. We are using a custom route serializer to convert the payload from the active route snapshot to our custom RouterParams. It carries a similar payload, but using a stripped down CUSTOM_ROUTER_NAVIGATION action in the app makes things more easy to mock out when testing.